cdap-ui/app/cdap/components/FieldLevelLineage/v2/FllTable/FllField.tsx

```diff
@@ -85,7 +90,7 @@ function FllField({ field, isActive, classes }: IFieldProps) {
       data-hovering={isHovering}
       data-target={isTarget}
     >
-      {field.name}
+      <span className={classes.fieldname}>{field.name}</span>
```

Comment:
did this work? span is an inline element, so it doesn't have a width. They shouldn't have an overflow

Reply:
It did work, but I don't know why...

Reply:
oh grid-row is setting display grid.. so all the children will not be inline anymore